.deployment.config.json

```diff
@@ -46,30 +46,58 @@
       }
     },
     {
-      "id": "deploy-headless-to-s3-version",
+      "id": "deploy-headless-minor-to-s3-version",
       "s3": {
         "bucket": "{terraform.infra.infra.bucket_binaries}",
-        "directory": "proda/StaticCDN/headless/v$[PRERELEASE]",
+        "directory": "proda/StaticCDN/headless/v$[MINOR]",
         "source": "packages/headless/dist/browser",
         "parameters": {
           "acl": "public-read"
         },
         "prd": {
-          "directory": "proda/StaticCDN/headless/v$[VERSION]"
+          "directory": "proda/StaticCDN/headless/v$[MINOR]"
         }
       }
     },
     {
-      "id": "deploy-atomic-to-s3-version",
+      "id": "deploy-headless-major-to-s3-version",
```

Comment:
File comparison is broken, I did not rename the atomic s3 upload, I copy and pasted headless's underneath its own and added `-major`.